src/initialize.ts

```diff
@@ -0,0 +1,23 @@
+import { KeyManager } from "./keymanager";
+import { PersistantStorage } from "./storage";
+import { User } from "./user";
+
+
+/**
+ * initliaze the sdk and return the user object
+ * @param keyManager  the key manager
+ * @param storage  the storage
+ * @returns the user object
+ */
+function initialize(keyManager: KeyManager, storage: PersistantStorage): User {
+  var _storage, _keyManager;
```

Comment:
what is the purpose of L13-19? I don't think it adds anything.

Reply:
do we do that? I though we always went through `userObj.storage.x`...

Reply:
right ill remove it